test/backend/datastores.mock.spec.js

```diff
@@ -32,7 +32,9 @@ describe('Datastore Mock:', function () {
     afterEach(() => {
         return closeTestServers(servers).then(() => {
             clearSettings('SETTINGS_PATH');
-            process.env.PLOTLY_CONNECTOR_STORAGE_PATH = storagePath;
+            if (storagePath) {
+                process.env.PLOTLY_CONNECTOR_STORAGE_PATH = storagePath;
```

Comment:
we should delete `process.env.PLOTLY_CONNECTOR_STORAGE_PATH` if it wasn't defined initially.

Reply:
By delete, you mean set it to `null`?

Reply:
> if it wasn't defined initially.

What does this means exactly? We do define it initially..

Reply:
Would be great if you could post a code snippet of what you are trying to say.. :)

Reply:
```js
if (storagePath) process.env.PLOTLY_CONNECTOR_STORAGE_PATH = storagePath;
else delete process.env.PLOTLY_CONNECTOR_STORAGE_PATH;
```